File: Php/poo/salary/library/person.php
<?php

    class person {

    private $name;
    private $lastName;
    private $identification;
    private $dayValue;
    private $daysWorked;

    public function __construct($name, $lastName, $identification, $dayValue, $daysWorked) {
        $this->name = $name;
        $this->lastName = $lastName;
        $this->identification = $identification;
        $this->dayValue = $dayValue;
        $this->daysWorked = $daysWorked;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getlastName(): string {
        return $this->lastName;
    }
    
    public function getIdentification(): int {
        return $this->identification;
    }

    public function getDayValue(): float {
        return $this->dayValue;
    }

    public function getDaysWorked(): int {
        return $this->daysWorked;
    }

    public function calculateSalary(): float {
        return $this->dayValue * $this->daysWorked;
    }
}

File: Java/poo/salary/library/person.java
package poo.salary.library;

public class person  {

    private String name;
    private String lastName;
    private int identification;
    private double dayValue;
    private int daysWorked;

    public person(String name, String lastName, int identification, double dayValue, int daysWorked) {
        this.name = name;
        this.lastName = lastName;
        this.identification = identification;
        this.dayValue = dayValue;
        this.daysWorked = daysWorked;
    }

    public String getName() {
        return this.name;
    }

    public String getLastName() {
        return this.lastName;
    }

    public int getIdentification() {
        return this.identification;
    }

    public double getDayValue() {
        return this.dayValue;
    }

    public int getDaysWorked() {
        return this.daysWorked;
    }

    public double calculateSalary() {
        return this.dayValue * this.daysWorked;
    }
}